test/test_core.rb

```diff
@@ -733,4 +733,24 @@ def test_deleted_inside_time_window
     assert_equal 0,
                  ParanoidTime.deleted_inside_time_window(3.minutes.from_now, 1.minute).count
   end
+
+  def test_configure_default_configuration
+    Rails.application.config.acts_as_paranoid_default_options = {
+      column: "test_column_name",
+      double_tap_destroys_fully: false,
+    }
+
+    temporary_paranoiac = Class.new(ActiveRecord::Base) do
+      acts_as_paranoid
```

Comment:
It is kind of important, that the class definition (rather this method call) is done after setting up the config, while I didn't want to change the behaviour of all other paranoiacs used in the tests. This is why I create this class inside of the test.